starter-code/javascript/chronometer.js

```diff
@@ -1,10 +1,79 @@
 class Chronometer {
-  constructor() {}
-  // startClick() {}
-  // getMinutes() {}
-  // getSeconds() {}
-  // twoDigitsNumber() {}
-  // stopClick() {}
-  // resetClick() {}
-  // splitClick() {}
+
+    constructor() {
+        this.currentTime = 0;
+        this.Milisecond = 0;
+        this.miliIntervalId = 0;
+        this.intervalId = 0;
+        this.status = "stopped";
```

Comment:
No need of this extra variable. You can track if the watch is running or not just by checking if interval id is null or not after u clearinterval.